cogs/normal/dice-normal.py

```diff
@@ -20,32 +20,60 @@ def __init__(self, bot):
     @commands.command(
         name="roll",
         description="command for rolling",
-        aliases=['r'],
+        aliases=["r"],
     )
     async def roll(self, ctx, *args):
-        if (len(args) < 1):
+        if len(args) < 1:
             await ctx.send("Please input a roll argument")
             return
 
         print(self.resolve_bool_roll(args))
         message = "<@{}> ".format(ctx.author.id) + self.resolve_bool_roll(args)
         await ctx.send(message)
 
+    @commands.command(
+        name="||roll",
+        description="command for rolling, but spoiler-ed",
+        aliases=["||r"],
+    )
+    async def roll(self, ctx, *args):
+        if len(args) < 1:
+            await ctx.send("Please input a roll argument")
+            return
+
+        print(self.resolve_bool_roll(args))
+        message = "<@{}> ".format(ctx.author.id) + self.resolve_bool_roll(args)
+        message = "||" + message.replace("||", "") + "||"
+        await ctx.send(message)
+
     @commands.Cog.listener()
     async def on_message(self, message):
-        if message.author == self.bot.user or message.content[0] == '`':
+        if message.author == self.bot.user or message.content[0] == "`":
```

Comment:
this line may usually cause harmless error.
in case someone share embed or picture, the message.content will actually be an empty string
```
if message.author == self.bot.user or len(message.content) == 0 or message.content[0] == "`":
```